### ISSUE TITLE
False positive unquoted resource title in case statements

### ISSUE THREAD

**Issue by dalen** [rodjek/puppet-lint#128]
The following code taken from the puppet style guide gives two warnings about unquoted resource titles in dust_bunny branch:

``` puppet
case $operatingsystem {
  centos: {
    $version = '1.2.3'
  }
  solaris: {
    $version = '3.2.1'
  }
  default: {
    fail("Module ${module_name} is not supported on ${operatingsystem}")
  }
}
```
